src/layout/styles/_dropper.scss

```diff
@@ -79,4 +65,134 @@
       height: 30%;
     }
   }
+
+  &[data-drop=horizontal-scroll] {
+    &::after {
+      top: 30%;
+      left: 20%;
+      height: 40%;
+      width: 20%;
+    }
+  }
+
+  &[data-drop=vertical-scroll] {
+    &::after {
+      top: 30%;
+      left: 60%;
+      height: 40%;
+      width: 20%;
+    }
+  }
+
+  &[data-drop=center] {
+    &::after {
+      top: 30%;
+      left: 40%;
+      height: 40%;
+      width: 20%;
+    }
+  }
+
+  > .drop-locations-overlay {
+    display: none;
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    left: 0;
+    top: 0;
+    background-color: rgba(255, 255, 255, .8);
+
+    .top-bottom,
+    .left-right,
+    .center {
+      position: absolute;
+
+      &::before,
+      &::after {
+        content: '';
+        display: block;
+        background-repeat: no-repeat;
+        background-position: 50%;
+        background-size: 5em;
+      }
+    }
+
+    .top-bottom,
```

Comment:
please avoid to many css classes, especially with such a generic name e.g `center` I cannot predict if there will be some interference with other styles. In this setup I would go with, first, nth-child, last since you exactly know the setup already.